src/OSPSuite.Infrastructure.Import/Services/SimulationResultsImporter.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using OSPSuite.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Services;
+using OSPSuite.Utility.Events;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Infrastructure.Import.Services
```

Comment:
ALso 100% comes from PKSim with the exception of one check that has been moved  out and will need to be done in PKsim. (actually not sure if this is required). I have created an issue for htat